src/github/listeners.ts

```diff
@@ -41,7 +41,12 @@ export async function onPush(
 
   const repoName = repository.full_name;
   const fileLoader = createGithubFileLoader(context.github, ref, repoName);
-  const config = await discoverConfig(fileLoader);
+  let config = null;
+  try {
+    config = await discoverConfig(fileLoader);
+  } catch (err) {
+    context.log.error(err);
```

Comment:
I don't think we should continue to lint if the configuration failed to load. Let's fail fast here, and later on we can save the error in the DB and show it on web UI.

Reply:
gotcha